cfme/tests/automate/test_common_methods.py

```diff
@@ -320,3 +321,60 @@ def _finalize():
         assert provision_request.is_succeeded(
             method="ui"
         ), f"Provisioning failed: {provision_request.row.last_message.text}"
+
+
+@pytest.fixture(scope="function")
+def vm_folder(provider):
+    """Create Vm folder on VMWare provider"""
+    folder = provider.mgmt.create_folder(fauxfactory.gen_alpha())
```

Comment:
This should be prefixed with something meaningful. Please use some `context+fauxfactory.gen_alpha()` 
As a VMware admin I would like to know who is creating the folders.